### ISSUE TITLE
provide pre-built releases of xhyve

### ISSUE THREAD

**Issue by philips** [coreos/coreos-xhyve#2]
make it easy for people to download a single archive with CoreOS and xhyve bundled together

**Comment by polvi** [comment]
we should make a brew for it

**Comment by philips** [comment]
@polvi agreed homebrew should have it. It would be cool to have a single archive with everything you need for less sophisticated users.

**Comment by bradgessler** [comment]
:+1:

**Comment by rimusz** [comment]
@philips @polvi I have made such App which has everything prebuilt and also is Status Bar App, where from the menu you can control boot, halt and etc. It's only external dependency is iTerm2.
https://github.com/rimusz/coreos-xhyve-ui

**Comment by AntonioMeireles** [comment]
@rimusz really GREAT work! (and thanks for the mention of my humble hacks in the credits)

**Comment by rimusz** [comment]
@AntonioMeireles thanks
I had to credit you, for your awesome work there :)
